### PR TITLE
Sterilizes the codebase - Pt.4 Removes putting collars on Humans

### PR DESCRIPTION
**Pull request by JamieD1**
Removes putting collars on people

:cl:  
rscdel: Removes putting collars on Humans
/:cl:

### CODE REVIEW

**Comment by Sgtdevel** [comment]
Wait does this mean we can still collar lizards why not just remove the feature completely?

**Comment by Smudged** [comment]
Collars can also be construted as Chockers, and we have rules against ERP.

**Comment by VaelophisNyx** [comment]
> Collars can also be construted as Chockers, and we have rules against ERP.

clothing is not ERP. If you want to make this argument, remove the sexy mime outfit and ladies (kinky) underwear.

**Comment by Hislyna** [comment]
I'd say don't make it felinid exclusive but allow everyone to wear them instead of removing it altogether

**Comment by Smudged** [comment]
> > Collars can also be construted as Chockers, and we have rules against ERP.
> 
> clothing is not ERP. If you want to make this argument, remove the sexy mime outfit and ladies (kinky) underwear.

Just to make sure, I don't want collars to be removed.

**Comment by maxion12345** [comment]
does seem pretty fetishy and beyond the realm of "hey this jumpskirt goes well with this scarf" and into "uwu what's this notices your [REDACTED] [DATA EXPUNGED]"

**Comment by JamieD1** [comment]
Tell me a reason why it should be in the game though.

**Comment by Georgeur** [comment]
I like the noise it makes when I walk around, don't have any justification for it than that.

**Comment by Marmio64** [comment]
> I like the noise it makes when I walk around, don't have any justification for it than that.

surely we could just add some bell boots or a jingly elf hat that makes the sound without needing to put pet collars on people

**Comment by Hislyna** [comment]
> Tell me a reason why it should be in the game though.

Personally I believe it is because it is just a clothing item that quite a few people wear in public. Make it wearable by everyone and there shouldn't be a problem. If its too much of a problem cause its a "pet collar", make a duplicate item and just call it a collar

**Comment by SomeguyManperson** [comment]
I need this feature to assert dominance over the lower classes

**Comment by Sniblet** [comment]
Now this is fetish content
Never had anything creepy happen while wearing it though

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -181,7 +181,7 @@
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
 	var/mob/living/carbon/C = M
-	if(C && ishumanbasic(C) && !iscatperson(C))
+	if(C && ishuman(C))
```
Comment (JamieD1) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
	if(C && !islizard(C))
```

**Pull request merged**